src/github/githubRepository.ts

```diff
@@ -36,15 +37,20 @@ export class GitHubRepository implements IGitHubRepository {
 	constructor(public remote: Remote, private readonly _credentialStore: CredentialStore) {
 	}
 
-	async resolveRemote(): Promise<void> {
-		try {
-			const { octokit, remote } = await this.ensure();
-			const { data } = await octokit.repos.get({
+	async getMetadata(): Promise<any> {

```

Comment:
in master there's a change to bump the version of the `octokit/rest` library, which adds return type information, so it should be easy to create an interface that extends that type to have `currentUser`

Reply:
agree and ideally we can have `currentUser` in `@octokit/rest` typings as well.